### PR TITLE
Add CompareMerge v2.03x

### PR DESCRIPTION
**Pull request by 17number**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/comparemerge.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'comparemerge' do
+  version '2.03x'
+  sha256 'ed6f53dd59481868ff29dc1d8800f5a821d918ce7fee1677f450bf302f23025c'
+
+  url "https://downloads.sourceforge.net/comparemergenosandbox/CompareMerge%20#{version}.zip"
```
Comment (commitay) [inline Casks/comparemerge.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)

Example:
https://github.com/caskroom/homebrew-cask/blob/1a322e32b7cd4383dfb154dcf0c194c4ae94c09d/Casks/psi.rb#L6-L9

File: Casks/comparemerge.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  name 'CompareMerge'
+  homepage 'https://sourceforge.net/projects/comparemergenosandbox/'
+
+  app 'CompareMerge 2.03x/CompareMerge 2.03x.app'
```
Comment (commitay) [inline Casks/comparemerge.rb]: Use `#{version}`  here

**Review by commitay — DISMISSED** [review]

**Review by commitay — APPROVED** [review]

**Comment by 17number** [comment]
Thank you for review and fix. It seems like I do not have to do anything, but should I do something?

P.S. I'm not good at English, so please let me know if there is inappropriate expression.

**Comment by commitay** [comment]
Thanks for the PR!

**Pull request merged**